docs/reference/troubleshooting/common-issues/red-yellow-cluster-status.asciidoc

```diff
@@ -215,12 +212,39 @@ watermark or set it to an explicit byte value.
 PUT _cluster/settings
 {
   "persistent": {
-    "cluster.routing.allocation.disk.watermark.low": "30gb"
+    "cluster.routing.allocation.disk.watermark.low": "90%",
+    "cluster.routing.allocation.disk.watermark.high": "95%"
   }
 }
 ----
 // TEST[s/"30gb"/null/]
```

Comment:
```suggestion
// TEST[s/"90%"/null/]
```

This is telling the test to substitute a value in the example. Because the example value was changed, but not the test, the test is failing.